help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -0,0 +1,153 @@
+
+Magnite Streaming destinations can receive the following identifier sources from the Adobe CDP. Please note all of the following identity sources can map to the Magnite device_id target identifier:
+
+| Identity Source             | Description                                                                                      | Consideration                                                                         |
+|:--------------------------- |:------------------------------------------------------------------------------------------------ |:------------------------------------------------------------------------------------- |
+| TV Device IDs               | Identifiers for advertising across Over-the-top (OTT)/ Connected TV (CTV) devices and platforms. | These IDs may be referenced as a custom identity source for the user.                 |
+| MAIDs                       | Google Advertising ID (GAID) or Apple ID for Advertisers (IDFA).                                 | These IDs are generally captured in the GAID and IDFA source namespaces for the user. |
+| Publisher Provided User Ids | This is generally a first-party, cross-device identifier that is included in the ad request.     | This may be referenced as a custom identity source for the user.                      |
+
+## Setting Up The Destinations
+
+Once your destination usage has been approved and Magnite has shared your credentials, please follow the below steps to authenticate, map, and share data.
+
+### **Authenticate to destination**
+
+Locate the Magnite Streaming destinations in the Adobe Experience catalog. Click the additional options button (\...) and then configure the destination.
+
+To authenticate to the destination, fill in the required fields and select Connect to the destination.
+
+![destination configuration auth fields unfilled](../../assets/catalog/advertising/magnite/destination-realtime-config-auth-unfilled.png)
+
+If you have an existing account, you can locate it by changing the Account type option to Existing account.
+
+### **Fill in destination details**
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required. You can then proceed by clicking Next.
+
+- **Name:** A name by which you will recognize this destination in the
+  future.
+
+- **Description:** A description that will help you identify this
+  destination in the future.
+
+![destination configuration auth fields filled](../../assets/catalog/advertising/magnite/destination-realtime-config-auth-filled.png)
+
+You can optionally select any relevant data governance policies. Data Export is generally used for the Magnite destinations.
+
+Once done, click the Create button.
+
+![Optional governance policy and enforcement actions](../../assets/catalog/advertising/magnite/destination-realtime-config-grouping-policy.png)
+
+### 
+
+### **Activate audiences to the destination**
```

Comment:
We have activation guides for all types of destinations, and the links are in the template:https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template#activate.

Do not remove those sections. Only remove what doesn't apply to this destination. This specific destination is a streaming one, so leave the entry for the streaming destination.

There is no need to explain the activation process here, since you're linking to the actual activation guide, so please remove the activation instructions and only leave mapping considerations, if needed.

Reply:
Updated